Scripts/RemediationScripts/Readme.md

```diff
@@ -651,6 +651,31 @@ Yes
 ___
 
 
+## 32. Azure_FrontDoor_CDNProfile_NetSec_Enable_WAF_Configuration
+
+### Display Name
+WAF Policy should be configured on Endpoints in Front Door
+
+### Link to Bulk Remediation Script (BRS)
+You can Configure WAF Policy on Front Door CDN using below BRS:  <br/>
+[Remediate-ConfigureWAFPolicyForFrontDoorCDN](Remediate-ConfigureWAFPolicyForFrontDoorCDN.ps1) <br/>
+You can enable State of WAF Policy configured on Front Door using below BRS:  <br/>
+[Remediate-EnableWAFPolicyForFrontDoorCDN](Remediate-EnableWAFPolicyForFrontDoorCDN.ps1) <br/>
```

Comment:
I don't see this BRS in current PR, do we have separate PR to cover that?

Reply:
[Remediate-ConfigureWAFPolicyForFrontDoorCDN] is already merged by Richa